scripts/superdesk/editor2/editor.ctrl.js

```diff
@@ -352,7 +353,7 @@ function SdTextEditorController(_, EMBED_PROVIDERS, $timeout, $element, editor,
         * Compute an id for the block with its content and its position.
         * Used as `track by` value, it allows the blocks to be well rendered.
         */
-        generateBlockId: function(block) {
+        generateBlockId: function(block, i) {
             function hashCode(string) {
```

Comment:
This should be renamed to at least `str` imo. I am amazed that `string` is not a reserved JS word.